selfdrive/loggerd/loggerd.cc

```diff
@@ -70,7 +69,8 @@ LogCameraInfo cameras_logged[LOG_CAMERA_ID_MAX] = {
     .downscale = false,
     .has_qcamera = true
   },
-  [LOG_CAMERA_ID_DCAMERA] = {
+#if defined(QCOM) || defined(QCOM2)
```

Comment:
we should remove these ifdefs, loggerd shouldn't really care about this stuff. should be handled automatically based on what kind of frame it gets

Reply:
if so, The code will be more complicated than it is now.